src/components/basic/badge/Badge.stories.tsx

```diff
@@ -0,0 +1,68 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { Badge, BadgeComponentProps } from './Badge';
+import { colorArgType } from '../../../docs/utils';
+
+const meta = {
+	component: Badge,
+	argTypes: {
+		color: colorArgType,
+		backgroundColor: colorArgType
+	}
+} satisfies Meta<BadgeComponentProps>;
```

Comment:
We can use typeof Badge instead of BadgeComponentProps, which should remain an internal type (by exporting it without changing the index.ts, the type is now public in the library)